docs/uninstall.md

```diff
@@ -107,7 +107,8 @@ The recommended steps to uninstall the OpenEBS cluster gracefully is as follows.
   ```
   
   If present, remove the finalizer entry from the corresponding BD and then delete it.
-
+  
+  After removing a pvc, we can still find a directory `/var/openebs/` having some of the volume sub-directory to store metadata info related to volume, for example `shared-pvc-69adec76-665e-46bc-b957-2b2f58338429-target` with no content. This can be removed manually once you successfully uninstall the OpenEBS cluster.
```

Comment:
Thanks @Bharat123rox  for the changes. Can you please make the following suggested changes?
1. pvc -> PVC
2.  we can still find a directory `/var/openebs/` -> you may find a directory `/var/openebs/` on the nodes where the replica was created
3. related to volume -> related to the volume
4. removed manually -> removed manually using `rm -rf` command

Reply:
@ranjithwingrider Updated